tests/unit/test_celery.py

```
@@ -341,6 +341,60 @@ def test_run_scan_celery_task(self):
+
+		with patch.object(app.backend, 'incr', return_value=42, create=True) as mock_incr:
+			self.assertEqual(bump_worker_loss_count('task-abc'), 42)
+			mock_incr.assert_called_once()
+
+	def test_bump_worker_loss_count_disabled_without_kv(self):
+		"""Returns 0 (cap disabled) on backends with neither incr nor get/set (db/RPC)."""
+		from unittest.mock import patch
+		from secator.celery import app, bump_worker_loss_count
+
+		with patch.object(app.backend, 'incr', side_effect=NotImplementedError, create=True), \
+				patch.object(app.backend, 'get', side_effect=NotImplementedError, create=True):
```

Flake8 (7.3.0)

[error] continuation line over-indented for visual indent

(E127)